server/data/orchestrationApiTypes.ts

```diff
@@ -16,7 +16,7 @@ export type CreateApplicationDto = components['schemas']['CreateApplicationDto']
 
 export type PrisonDto = components['schemas']['PrisonDto']
 
-export type PrisonerInfoDto = components['schemas']['PrisonerInfoDto']
+export type BookerPrisonerInfoDto = components['schemas']['BookerPrisonerInfoDto']
```

Comment:
Most of the changes in this PR are a result of this DTO changing name and structure. I've kept the changes to the data layer; in the service layer now fully define our own `Prisoner` type with the data we need and a better structure.